lib/typing/columns/columns.go

```diff
@@ -31,16 +31,12 @@ type Column struct {
 	backfilled bool
 }
 
-func (c *Column) PrimaryKey() bool {
-	return c.primaryKey
+func (c Column) IsValid() bool {
+	return c.KindDetails != typing.Invalid
 }
 
-func (c *Column) ShouldSkip() bool {
```

Comment:
Removed this function to standardize on `IsValid`.

2 reasons:

1. We should not need to have `Column` be a ptr, removing it allows us to remove the need to check if it's a ptr.
2. Rest of the logic is the same as `IsValid`